recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend

```diff
@@ -8,6 +8,16 @@ XT_GUESTS_BUILD ?= "doma"
 XT_GUESTS_INSTALL ?= "doma"
 
 python __anonymous () {
+    machine = d.getVar('MACHINE', True)
+    not_supported_machines = ["salvator-x-m3", "salvator-xs-h3", "h3ulcb", "m3ulcb", "salvator-x-h3"]
+    if machine in not_supported_machines:
+        warning = 'The machine {} is not supported'.format(machine) 
+        bb.warn(warning)
+        warning = "Use the branch 1.11/5375571-7.1.0-10.0.0_r3-xt0.1 of ssh://git@gitpct.epam.com/epmd-aepr to build the android , \
+                   and 1.11/5516664 to build domd and domu."
+        bb.warn(warning)
+        bb.fatal("Use branch xen branch RELEASE-4.13.0-xt0.1.prod-devel for xen-troops/xen project to support.")
+
```

Comment:
As we have already discussed, wouldn't it be better to point to prod-devel branch to use, which comprises all required changes to successfully build for these machines?

Reply:
at this moment we do not have a separate branch, and I am not sure if it is required, because, in the case, it is necessary support 2 branches.
I think it is enough to mention about the pvr branches to build the prod-devel to support the machines.